src/applications/disability-benefits/all-claims/tests/validations.unit.spec.js

```diff
@@ -0,0 +1,48 @@
+import sinon from 'sinon';
+import { expect } from 'chai';
+
+import { isValidYear } from '../validations';
+
+describe('526 All Claims validations', () => {
+  describe('isValidYear', () => {
+    it('should add an error if the year is not a number', () => {
+      const err = {
+        addError: sinon.spy(),
+      };
+      isValidYear(err, 'asdf');
+      expect(err.addError.called).to.be.true;
+    });
+
+    it('should add an error if the year contains more than just four digits', () => {
+      const err = {
+        addError: sinon.spy(),
+      };
+      isValidYear(err, '1990asdf');
+      expect(err.addError.called).to.be.true;
+    });
+
+    it('should add an error if the year is less than 1900', () => {
+      const err = {
+        addError: sinon.spy(),
+      };
+      isValidYear(err, '1899');
+      expect(err.addError.called).to.be.true;
+    });
+
+    it('should add an error if the year is more than 3000', () => {
+      const err = {
+        addError: sinon.spy(),
+      };
+      isValidYear(err, '3001');
+      expect(err.addError.called).to.be.true;
+    });
+
+    it('should not add an error if the year is between 1900 and 3000', () => {
```

Comment:
Seems kind of broad. Are you sure that the separation pay date doesn't have to be in the past?

Reply:
I...am not sure. That would make sense to me, but it wasn't in the requirements.

@djmunoz1 Can you confirm (or ask whomever can confirm)? Should the separation pay date have a year that's either the current year or in the past? Should it also be after the first period of service start date or the last PoS end date?